### PR TITLE
[Example] Provide more results with various numerical embeddings in AutoMM on tabular examples

### PR DESCRIPTION
**Pull request by yiqings**
*Issue #, if available:*

*Description of changes:*

We provide more (ablation) results with various numerical embeddings in AutoMM on tabular examples.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/automm/tabular_dl/README.md [hunk 1]
```diff
@@ -97,7 +96,35 @@ unzip tabular_example_result.zip
...
+metrics | rmse | acc | acc | acc | acc | acc | acc | rmse | acc | rmse | rmse
+["linear" ] | 0.482 | 0.859 | 0.379 | 0.721 | 0.726 | 0.949 | RuntimeError | 8.891 | 0.963 | 0.769 | 0.761
+["linear", "relu"] | 0.477 | 0.859 | 0.370 | 0.721 | 0.726 | 0.951 | RuntimeError | 8.953 | 0.967 | 0.772 | 0.757
+["linear", "leaky_relu"] | 0.473 | 0.858 | 0.370 | 0.722 | 0.725 | 0.947 | RuntimeError | 8.915 | 0.965 | 0.771 | 0.776
+["linear", "relu", "linear"] | 0.468 | 0.858 | 0.374 | 0.721 | 0.723 | 0.951 | RuntimeError | 8.941 | 0.965 | 0.769\* | 0.770
+["positional", "linear"] | 0.467 | 0.864 | 0.347 | 0.694 | 0.709 | 0.951 | RuntimeError | 9.120 | 0.967 | 0.773\* | 0.761
+
+\* denotes adjusting `env.per_gpu_batch_size` from `128` to `64` to support runing a larger model on our device.
+
+Download our results:
```
Comment (sxjscience) [inline examples/automm/tabular_dl/README.md:118]: Let's attach the results as another column of the table.

**Comment by sxjscience** [comment]
It seems that the embedding methods do not make much difference. Do you think that we may pick a search space for the lr and train the models for all the learning rates in the search space? We can also try the recent HPO feature in AutoGluon.

**Comment by szha** [comment]
Job PR-1903-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1903/1/index.html

**Comment by yiqings** [comment]
> It seems that the embedding methods do not make much difference. Do you think that we may pick a search space for the lr and train the models for all the learning rates in the search space? We can also try the recent HPO feature in AutoGluon.

For numerical embeddings:

Yes, it does not achieve a consistent improvement to all datasets. But it does boost some datasets. For example from 0.482 RMSE to 0.467 RMSE on CA, 0.859 ACC to 0.864 ACC on AD.

For HPO:

That's a great suggestion. We can have a try.

Based on our previous experiments, `lr` does not take dominance on the 10 tabular datasets (ep is excluded as it is too large to run with): We have tried using a smaller `lr=1e-5` or a constant learning rate scheduler (with end learning rate set to 0), where both do not achieve any significant improvement/decrease. 

Actually, the network architectures itself provide more important hyper-parameters needed to search. In the original FT-Transformer paper, for example, the authors tuned `attention_dropout, d_token, ffn_dropout, prenormalization` on some datasets. Thus, we can probably first consider trying it on FT_Transformer before digging into the numerical embeddings.

**Review by sxjscience — APPROVED** [review]
LGTM

**Comment by szha** [comment]
Job PR-1903-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1903/2/index.html
